src/Auth/Authenticator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+interface Authenticator
+{
+    public function login(Identifiable $identifiable): void;
+
+    public function logout(): void;
+
+    public function user(): Identifiable|array|null;
```

Comment:
What is this method used for?

Reply:
This is to return the identifiable data from the session